src/app/index.ts

```diff
@@ -320,7 +329,22 @@ module.exports = class TheiaExtension extends Base {
                 this.extensionPath('README.md'),
                 { params: this.params }
             );
-        }
+            this.fs.copyTpl(
+                this.templatePath('widget/__tests__/widget.test.ts'),
```

Comment:
We should follow the format of other tests found in the upstream repo (drop `__tests__` as a name).